server.js

```diff
@@ -1,25 +1,55 @@
 // Require the Express module (https://npmjs.com/package/express)
-
+var express = require("express");
 
 // Create a new express application instance by calling `express()`
-
+var app = express();
 
 // Serve files in the 'public' directory with Express's built-in static file server
-
+app.use(express.static('public'));
 
 // Create a Counter class that will be used to create counter objects
 // See the full description in README.md
 
 
+function Counter() {
+    this.pet = {
```

Comment:
It would be better to initialize this property with an empty object. Don't add the 'puppies' and 'kittens' keys here.